src/azul/collections.py

```diff
@@ -91,8 +91,9 @@ class deep_dict_merge[K, V](dict):
     {}
     """
 
-    def __init__(self, *maps: Mapping[K, V]):
+    def __init__(self, *maps: Mapping[K, V], override: bool = False):
```

Comment:
It should be understood that this new parameter needs to be documented and covered by doctests.